### DISCUSSION TITLE
Auto start container after server restart

### DISCUSSION THREAD

**Discussion by Ejayz** [louislam/dockge#230] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I was wondering if there is a way to auto start container if i reboot my server? Either selection of conatiner or all of container that i created .

### 🐻 Dockge Version

Version: 1.2.0

### 💻 Operating System and Arch

Ubuntu 22.04.3 LTS

### 🌐 Browser

Chrome

### 🐋 Docker Version

Docker version 24.0.7, build afdd53b

### 🟩 NodeJS Version

v21.1.0

**Comment by louislam** [comment, marked as answer]
it's a docker question.

You have to set the `restart policy` to `unless-stopped` or `always`

**Reply by Ejayz** [reply]
hi . This is one of the config on my dockler container .
version: "3.3"
services:
  mariadb:
    container_name: MariaDB
    restart: unless-stopped
    environment:
      - MYSQL_ROOT_PASSWORD=somepassword
    ports:
      - 3306:3306
    image: docker.io/library/mariadb:10.3
networks: {}

is it the restart:unless-stopped? which is the recommended one ? it doesnt auto start my containers which i tried .
